src/system_database.ts

```diff
@@ -51,9 +53,15 @@ export class PostgresSystemDatabase implements SystemDatabase {
     this.pool = new Pool(poolConfig);
   }
 
-  async init() {
+  async init(recreateSystemDB: boolean) {
     const pgSystemClient = new Client(this.pgPoolConfig);
     await pgSystemClient.connect();
+
+    // If recreate is set to true, drop the existing system database.
+    if (recreateSystemDB) {
```

Comment:
Let's get rid of this and just have helper functions in the test--I don't want a nuke button in the source code.

Reply:
Okay makes sense.